consumer-configuration/README.adoc

```diff
@@ -0,0 +1,347 @@
+
+[id="proc-editing-consumer-group-properties_{context}"]
+== Reviewing consumer group properties in {product}
+
+[role="_abstract"]
+Use the {product} web console to select a consumer group and review the consumer properties.
+You can look at the consumer groups associated with a Kafka instance or a specific topic of the instance.
+
+As an alternative to using the {product} web console, you can use the `rhoas` command-line interface (CLI) to return information on a consumer group, as shown in the following example command:
+
+.Example CLI command to return consumer group information
+[source]
+----
+rhoas kafka consumergroup describe [flags]
+----
+
+For a list of consumer group commands that you can use with the CLI, see the {base-url-cli}{rhoas-cli-ref-url}[_CLI command reference (rhoas)_^].
+
+If you are using Kafka bin scripts, you can use the `kafka-consumer-groups.sh` tool to provide the same information:
+[source,subs="+quotes,+attributes"]
+----
+bin/kafka-consumer-groups.sh --bootstrap-server __<bootstrap_server>__ --describe --group my-group-id
+----
+
+The `<bootstrap_server>` is the bootstrap server endpoint for your service account.
+You copy this information for the Kafka instance in the {product} web console by selecting the options menu (three vertical dots) and clicking *View connection information*.
+
+ifndef::community[]
+NOTE: The Kafka bin scripts are part of the open source community version of Apache Kafka. The bin scripts are not a part of {product} and are therefore not supported by Red Hat.
+endif::[]
+
+.Prerequisites
+* You are logged in to the {product} web console.
+* You have created a Kafka instance with at least one Kafka topic in {product}.
+* Consumer client applications connected to the Kafka instance have a consumer group ID.
+
+.Procedure
+. In the {product} web console, go to *Streams for Apache Kafka* > *Kafka Instances* and click the name of the Kafka instance that contains the consumer groups that you want to review.
++
+Alternatively, select a consumer group for a specific topic.
++
+--
+.. In the {product} web console, go to *Streams for Apache Kafka* > *Kafka Instances* and click the name of the Kafka instance that contains the topic.
+.. In the *Topics* page, click the name of the topic.
+--
+
+. In the *Consumer groups* page, click the name of the consumer group that you want to review.
++
+Alternatively, select the options icon (three vertical dots) for the relevant consumer group and click *View partition offsets*.
+
+[role="_additional-resources"]
+.Additional resources
+* {base-url}{rhoas-cli-url}[_Getting started with the `rhoas` CLI_^]
+* {base-url-cli}{rhoas-cli-ref-url}[_CLI command reference (rhoas)_^]
+
+[id="ref-supported-consumer-group-properties_{context}"]
+== Consumer group properties presented in {product}
+
+[role="_abstract"]
+The following consumer group properties are presented in {product}.
```

Comment:
Strictly opinion, so disregard as needed, but I'd say for the heading "...properties in {product}" and nix "presented". And also I tend to go with "displayed", so "properties are displayed in ...." for the first line, just because presented seems less common and can have a meaning of a presentation rather than a UI display. Most instances in the IBM SG use "display" and "displayed". But if you have a strong opinion otherwise, understood.

Reply:
Changed.